src/components/ResourceItem/ResourceItem.scss

```
@@ -343,3 +335,21 @@ $resource-list-item-variables: (
     background-color: var(--p-surface-selected-pressed);
   }
 }
+
+.ListItem {
+  position: relative;
+  @include focus-ring($border-width: rem(0px), $size: 'wide');
```

```suggestion
  @include focus-ring($size: 'wide');
```

Shouldn't need to provide the border width because the default is rem(0px)